src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmRepository.java

```diff
@@ -19,25 +19,14 @@
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
  */
 @Repository
-public interface IgmRepository extends CassandraRepository<IgmEntity, IgmEntityKey> {
+public interface IgmRepository extends JpaRepository<IgmEntity, IgmEntityKey> {
 
-    @Query("SELECT * FROM merge_igm WHERE processUuid = :processUuid")
-    List<IgmEntity> findByProcessUuid(UUID processUuid);
+    List<IgmEntity> findByKeyProcessUuid(UUID processUuid);
 
-    @Query("SELECT * FROM merge_igm WHERE processUuid = :processUuid AND date = :date")
-    List<IgmEntity> findByProcessUuidAndDate(UUID processUuid, LocalDateTime date);
+    List<IgmEntity> findByKeyProcessUuidAndKeyDate(UUID processUuid, LocalDateTime date);
 
-    @Query("DELETE FROM merge_igm WHERE processUuid = :processUuid")
-    void deleteByProcessUuid(UUID processUuid);
+    @Modifying
+    void deleteByKeyProcessUuid(UUID processUuid);
```

Comment:
In the docs I see "The `@Modifying` annotation [...] is not applied on [...] queries derived from the method name". Are you sure you need this ? In other similar cases we have used `@Transactional` I think

Reply:
Corrected